ci/test-stable.sh

```diff
@@ -117,3 +117,10 @@ export CARGO_TOOLCHAIN=+"$rust_stable"
   set -x
   ci/localnet-sanity.sh -x
 )
+
+echo --- ci/run-sanity.sh
+export CARGO_TOOLCHAIN=+"$rust_stable"
```

Comment:
Move this export down two lines into the subshell `(..)` so it doesn't affect the current shell